Gifski/Gifski.swift

```diff
@@ -19,6 +20,8 @@ final class Gifski {
 				return "Invalid settings."
 			case .unreadableFile:
 				return "The selected file is no longer readable."
+			case let .notEnoughFrames(frameCount):
+				return "An animated GIF requires at least 2 frames. Your video only has \(frameCount) frame\(frameCount == 1 ? "" : "s")."
```

Comment:
I know this is not true, but it's easier to enforce 2 frames than having users being confused why their GIF is not playing because it only has 1 frame.

Reply:
I think `Your video contains 0 frames` is a better way to word it IMHO

Reply:
Agreed